psydac/feec/tests/test_pushforward.py

```diff
@@ -16,13 +16,6 @@ def test_basic_call():
     degree = [2, 2]
 
     # Mapping and physical domain
-    class CollelaMapping2D(Mapping):
-
-        _ldim = 2
-        _pdim = 2
-        _expressions = {'x': 'a * (x1 + eps / (2*pi) * sin(2*pi*x1) * sin(2*pi*x2))',
-                        'y': 'b * (x2 + eps / (2*pi) * sin(2*pi*x1) * sin(2*pi*x2))'}
-
     mapping = CollelaMapping2D('M', a=1, b=1, eps=.2)
```

Comment:
the `CollelaMapping2D` from `sympde` has `k1` and `k2` as parameters. This line is probably wrong.